NumberBaseball/NumberBaseball/Step1.swift

```diff
@@ -0,0 +1,62 @@
+import Foundation
+
+var computerRandomNumbers: Set<Int> = []
+var randomNumbers: Set<Int> = []
+
+func createComputerRandomNumbers() -> [Int] {
+    while computerRandomNumbers.count != 3 {
+        computerRandomNumbers.insert(Int.random(in: 1...9))
+    }
+    let computerRandomNumbers = Array(computerRandomNumbers)
+    return computerRandomNumbers
+}
+
+func createRandomNumbers() -> [Int] {
+    randomNumbers.removeAll()
+    while randomNumbers.count != 3 {
+        randomNumbers.insert(Int.random(in: 1...9))
+    }
+    let randomNumbers = Array(randomNumbers)
+    return randomNumbers
+}
+
+func judgeResult(computerRandomNumbersArray: [Int],randomNumbersArray: [Int]) -> [Int] {
+    var ball: Int = 0
+    var strike: Int = 0
+    var resultOfStrikeAndBall: [Int] = []
+    
+    for i in 0...2 {
+        if computerRandomNumbersArray[i] == randomNumbersArray[i] {
+            strike += 1
+        } else if computerRandomNumbersArray[i] != randomNumbersArray[i] && randomNumbersArray.contains(computerRandomNumbersArray[i]) {
+            ball += 1
+        }
+    }
+    resultOfStrikeAndBall.append(strike)
+    resultOfStrikeAndBall.append(ball)
+    return resultOfStrikeAndBall
+}
+
+func startGame() {
+    var remainCount: Int = 9
+    let computerRandomNumbersArray = createComputerRandomNumbers()
+    let randomNumbersArray = createRandomNumbers()
+    let judgeResult = judgeResult(computerRandomNumbersArray: computerRandomNumbersArray, randomNumbersArray: randomNumbersArray)
+    
+    while remainCount != 0 && judgeResult[0] != 3 {
```

Comment:
```suggestion
    while remainCount > 0 {
```
은 어떠신가요? 만약 remainCount 가 0 밑으로 떨어지게 된다면 무한루프의 위험이 존재할 것 같네요!

Reply:
수정했습니다!
혹시 이런식의 조건을 설정할 때 안전성 측면에서 ==나 != 연산자보다 <, <= 연산자를 사용하는게 좋을까요??

Reply:
한 조건에 대해서만 검사하고 싶을 때 ==나 != 연산자를 사용하게 되고 범위에 대해서 검사하고 싶을땐 범위 연산자를 사용하는 겁니다! ㅎㅎ